VectorService.md

```diff
@@ -0,0 +1,142 @@
+
+Requests can be sent in the form of GET or POST (form or json).
+
+## Request
+
+A request can be broken out into two distinct types identified through a parameter called `vector_type`. Available  `vector_type`'s are: **carotene_skills** and **semantic_fingerprint**.
+
+### Carotene Skills
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| carotene_id | String | True | A carotene ID as provided by our Job Classification services which uniquely identifies a job title.
+| carotene_version   | String | True | The version of carotene the ID is associated with.
+| skills | List | False | A list of skill objects.
+
+Each skill object is made up of a unique number/letter identifier provided as a String.
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| id | String | False | A Skill ID as provided from our Skills tagging service [here](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/Skills.md)
+
+#### CaroteneSkills Only Request Sample
+```
+{
+  "carotene_id": "11.0",
+  "carotene_version": "CAROTENEV3",
+  "skills": [
+    {
+      "id": "KSH9TO"
+    }
+  ],
+  "vector_types": ["carotene_skills"]
+}
+
+```
+
+### Semantic Fingerprint
+Semantic Fingerprint requests are processed by a deep learning model produced by Textkernel. The model can process both job and resume documents identified by the `document_type` param. If a document is provided the `document_type` is required and requests sent for this vector transformation will fail without it.
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| document_type | String | True | The type of document being sent. Accepted values are: **job** and **resume**.
+| document   | String | False | The base64 string representation of a file.
+| job_title | String | False | The job title associated with the job description document being sent.
+| job_requirements | String | False | The requirements associated with the job document.
+
+#### SemanticFingerprint Only Request Sample
+```
+{
+  "document": "amF2YQo=",
+  "job_title": "Engineer",
+  "job_requirements": "requirements for an engineer",
+  "document_type": "job",
+  "vector_types": ["semantic_fingerprint"]
+}
+```
+
+## All Vector Types Sample Request
+```
+{
+  "carotene_id": "11.0",
+  "carotene_version": "CAROTENEV3",
+  "skills": [
+    {
+      "id": "KSH9TO"
+    }
+  ],
+  "document": "amF2YQo=",
+  "job_title": "Engineer",
+  "job_requirements": "requirements for an engineer",
+  "document_type": "job",
+  "vector_types": ["carotene_skills", "semantic_fingerprint"]
+}
+```
+
+
+## Response Structure
+
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node. The response will contain each `vector_type` requested and an associated list of decimal numbers representing the computed vectors for the request.
+
```

Comment:
But response will not contain both If I am requesting only one of the algorithms right. Do we need to mention it?